src/server/server.rs

```diff
@@ -122,7 +129,8 @@ impl<T: RaftStoreRouter, S: StoreAddrResolver> Server<T, S> {
             conn_token_counter: FIRST_CUSTOM_TOKEN.as_usize(),
             store_tokens: HashMap::new(),
             store_resolving: HashSet::new(),
-            raft_router: raft_router,
+            raft_router: ch.router,
```

Comment:
`ServerChannel` is a structure to bundle the arguments. It doesn't provide a abstract layer to wrap the existing router.